src/testTrackConfig.js

```diff
@@ -37,15 +38,15 @@ var TestTrackConfig = {
   },
 
   getSplitRegistry: function() {
-    var rawRegistry = getConfig().splits;
+    const rawRegistry = getConfig().splits;
 
     if (!rawRegistry) {
       return new SplitRegistry(null);
     }
 
     if (!registry) {
-      var splits = Object.keys(rawRegistry).map(function(splitName) {
-        var rawSplit = rawRegistry[splitName];
+      const splits = Object.keys(rawRegistry).map(function(splitName) {
```

Comment:
```suggestion
      const splits = rawRegistry.map(({splitName, feature_gate, weights}) => new Split(splitName, feature_gate, weights));
```
         ¯\_(ツ)_/¯

Reply:
We could do 

```suggestion
      const splits = Object.entries(rawRegistry).map(function([splitName, split]) {
```
But you can't `map` plain Objects.